defaults/main.yml

```diff
@@ -1,10 +1,16 @@
-argus_radium_user: "{{ argus_user }}"
-argus_radium_group: "{{ argus_group }}"
+argus_radium_user: "{{ __argus_radium_user }}"
+argus_radium_user_home: "{{ __argus_radium_user_home }}"
+argus_radium_user_shell: "{{ __argus_radium_user_shell }}"
+argus_radium_user_comment: argus radium daemon
+argus_radium_group: "{{ __argus_radium_group }}"
 argus_radium_package: "{{ __argus_raduim_package }}"
-argus_radium_log_dir: "{{ argus_log_dir }}"
+argus_radium_log_dir: /var/log/argus
+argus_radium_log_dir_owner: "{{ argus_radium_user }}"
+argus_radium_log_dir_group: "{{ argus_radium_group }}"
+argus_radium_log_dir_mode: "0755"
 argus_radium_service: radium
 argus_radium_conf_dir: "{{ __argus_radium_conf_dir }}"
-argus_radium_conf_file: "{{ argus_radium_conf_dir }}/radium.conf"
-argus_radium_flags: "-f {{ argus_radium_conf_file }}"
+argus_radium_conf_file: "{{ __argus_radium_conf_dir }}/radium.conf"
```

Comment:
s/__argus_radium_conf_dir/argus_radium_conf_dir/
or users must modify argus_radium_conf_file when argus_radium_conf_dir is set.